### ISSUE TITLE
Ai-thinker on ESP-VA-SDK

### ISSUE THREAD

**Issue by tcpipchip** [kasiim/ESP-ADF-AI-thinker-support#1]
Hi @kasiim 

How are you my friend ?

Did you verify if your routines will works too on ESP-VA-SDK ?
https://github.com/espressif/esp-va-sdk/blob/master/README-Getting-Started.md
I want to use their amazon/google example with a AI-THINKER ESP32

Can you test, please ?
Got a lot of errors on ESP-VA-SDK
Looks that your routine only works with ESP-ADF-SDK.

**Comment by tcpipchip** [comment]
look
![image](https://user-images.githubusercontent.com/11488460/86512976-c1cb4380-bddc-11ea-8abf-ebf8e4b76dab.png)

perfect in the right side, using ESP-ADF-SDK,!

bad in left side, using the ESP-VA-SDK

only you to find out this problem! :)

**Comment by kasiim** [comment]
Hello,

Problem is that the VA-SDK has been developed with many more features than this repository offers.
To use AI-thinker board, more functionality must be provided by library. I.e DSP and LED drivers must be coded, probably most of them can be reused by example support packages from LyraT etc.
This repository only describes board and ESP32-A1S codec usage.

This is the reason you are getting errors.

In conclusion there is no easy fix for this problem.

**Comment by tcpipchip** [comment]
first,
Thanks to answer me! I am spending 40 hours without success! Making me crazy! Reaching a point to not use the AI-THINKER in my project!
But, can you transport the AMAZON/GOOGLE example to ESP-ADF-SDK or is too complex ?

**Comment by kasiim** [comment]
Sure, AI-THINKER boards unfortunately are not espressif official boards. So support for them is purely from community. It probably would take about a day or so to enable this support. Unfortunately providing this support is not my top priority at the moment. Might be in future though.

Have a good day!

**Comment by tcpipchip** [comment]
But, can be converted the AMAZON/GOOGLE example to ESP-ADF-SDK or is too complex ? (one more question :) )

**Comment by kasiim** [comment]
I think that there is a reason they made it seperate SDK. So my answer would be that easier solution is to customize this repository.

**Comment by tcpipchip** [comment]
Thank you so much!

**Comment by tcpipchip** [comment]
have you interest to add a new codec ? I have here a ESP32-WROOVER BOARD with a different codec...
